KitchenLib/src/Systems/AchievementUnlockSystem.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Kitchen;
+using KitchenLib.Achievements;
+using KitchenLib.Components;
+using KitchenLib.Utils;
+using KitchenMods;
+using Unity.Entities;
+using UnityEngine;
+
+namespace KitchenLib.Systems
+{
+	public class AchievementUnlockSystem : GameSystemBase, IModSystem
+	{
+		internal static AchievementUnlockSystem Instance;
+		protected override void OnUpdate()
+		{
+			if (Instance == null) Instance = this;
+			return;
+			if (Input.GetKeyDown(KeyCode.F))
+			{
+				AchievementsManager.GetManager("kitchenlib").UnlockAchievement("test");
+			}
+			if (Input.GetKeyDown(KeyCode.G))
+			{
+				AchievementsManager.GetManager("kitchenlib").UnlockAchievement("test2");
+			}
+			if (Input.GetKeyDown(KeyCode.H))
+			{
+				AchievementsManager.GetManager("kitchenlib").UnlockAchievement("test3");
+			}
+		}
```

Comment:
Would be better to comment out or remove altogether the tests as they're never run in the first place.